client/galaxy/scripts/components/Masthead.vue

```diff
@@ -0,0 +1,210 @@
+<template>
+        <div>
+            <b-navbar id="masthead" toggleable="lg" type="dark"
+                      role="navigation" aria-label="Main" class="justify-content-center">
+                <b-navbar-brand :href="brandLink" aria-label="homepage">
+                    <img alt="logo" class="navbar-brand-image" :src="brandImage"/>
+                    <span class="navbar-brand-title">{{ brandTitle }}</span>
+                </b-navbar-brand>
+
+                <b-navbar-nav>
+                    <template v-for="tab in tabs">
+                            <b-nav-item v-if="!tab.menu"
```

Comment:
This `b-nav-item` should be in its own subcomponent e.g. MastheadItem.vue, then the below class and property settings can be computed values. It will simplify and improve readibility of styling and configuring an individual navitem.